src/azure-cli/azure/cli/command_modules/alertsmanagement/_help.py

```diff
@@ -0,0 +1,429 @@
+  - name: --name
+    type: string
+    short-summary: Action Rule Name.
+  - name: --description
+    type: string
+    short-summary: Description Actions rules in a subscription.
+  - name: --status
+    type: string
+    short-summary: Status of Action Rule
+  - name: --scope
+    type: string
+    short-summary: Comma separated list of values
+  - name: --severity-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>}
+  - name: --monitor-service-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>}
+  - name: --monitor-condition
+    type: string
+    short-summary: Supported values – Fired, Resolved
+  - name: --target-resource-type-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>}
+  - name: --alert-ruleId-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>} For eg. Equals:ARM_ID_1,ARM_ID_2
+  - name: --description-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>} For eg. Contains:Test Alert
+  - name: --alert-context-condition
+    type: string
+    short-summary: Expected format - {<operation>:<comma separated list of values>} For eg. Contains:smartgroups
+  - name: --action-rule-type
+    type: string
+    short-summary: Action rule Type.
+  - name: --recurrence-type
+    type: string
+    short-summary: Specifies the duration when the suppression should be applied.
+  - name: --suppression-start-time
+    type: string
+    short-summary: Suppression Start Time. Format 12/09/2018 06:00:00 Should be mentioned in case of Reccurent Supression Schedule - Once, Daily, Weekly or Monthly
+  - name: --suppression-end-time
+    type: string
+    short-summary: Suppression End Time. Format 12/09/2018 06:00:00 Should be mentioned in case of Reccurent Supression Schedule - Once, Daily, Weekly or Monthly
+  - name: --recurrence-values
+    type: string
+    short-summary: Reccurent values, if applicable. In case of Weekly - 1,3,5 In case of Monthly - 16,24,28
+  - name: --action-group-id
+    type: string
+    short-summary: Action Group Id which is to be notified
+examples:
+  - name: Set action-rule on various parameters.
+    text: >
+        az alertsmanagement action-rule set --resource-group-name "test-rg" --name "Test-AR" --scope "/subscriptions/dd91de05-d791-4ceb-b6dc-988682dc7d72/resourceGroups/alertslab","/subscriptions/dd91de05-d791-4ceb-b6dc-988682dc7d72/resourceGroups/Test-VMs" --severity-condition "Equals:Sev0,Sev1" --monitor-condition "NotEquals:Resolved" --description "Test description" --status "Enabled" --action-rule-type "Suppression" --recurrence-type "Weekly" --suppression-start-time "06/26/2018 06:00:00" --suppression-end-time "07/27/2018 06:00:00" --recurrence-values 1,4,6
```

Comment:
Usually in examples we use "MyResourceGroup" as resource group name. Also alert name coulld be "MyAlert".
Also it's better not to include real subscription id "dd91de05-d791-4ceb-b6dc-988682dc7d72"